lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -555,7 +541,7 @@ let fetch = { private ? false, fetchSubmodules ? false, owner, repo, rev, sha256
     inherit owner repo rev sha256 fetchSubmodules private;
   };
   json = builtins.fromJSON (builtins.readFile ./github.json);
-in if builtins.pathExists ./local then ./local else fetch json
+in fetch json
```

Comment:
This causes `Thunk update: ReadThunkError_UnrecognizedThunk` when trying to update a thunk from before this change. I think `ThunkSpec`s need to be immutable.